stubs/ext-ds.stub

```diff
@@ -593,8 +599,9 @@ final class Set implements ArrayAccess, Collection, Traversable
 	}
 
 	/**
-	 * @param Set<TValue> $set
-	 * @return Set<TValue>
+	 * @template TValue2
+	 * @param Set<TValue2> $set
+	 * @return Set<TValue|TValue2>
```

Comment:
Actually this is wrong. `diff` only returns values from the first set that are not in the second set. So the result won't contain `TValue2`. Same goes for `Map::diff()`